utils/commons.js

```diff
@@ -27,6 +30,14 @@ class Commons{
     }
   }
 
+  // these files exist in cloud automation repo, check there for details
+  static get userAccessFiles() {
+    return {
+      newUserAccessFile1: 'test1_user.yaml',
```

Comment:
file1 is never used
Also we could change the logic for this - from our previous discussion:
"we need a function that modifies the backed up YAML with whats in the test2 file (changed access for test users) and then we can run useryaml job with that file"
(see `setUserYaml` function)
Maybe a refactoring idea for later~~

Reply:
`backupUserYaml()` is not used for now but let's keep it in case we want to improve this